### DISCUSSION TITLE
Executale (.exe) of 160 MB

### DISCUSSION THREAD

**Discussion by gerardo-grelle** [community/community#51250] [New to GitHub]
I'd like to upload an executable (.exe) of 160 MB in my repository. What can I do?

**Comment by matej-kaska** [comment]
You need to use git lfs (Git Large File Storage). Then you can upload (push) maximum 2 GB file with Free and Pro Tier of Github account.

How to do it: https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage
Limits of repositories on GitHub: https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage

**Comment by airtower-luna** [comment]
Most likely you shouldn't, generally a repository should have *only* source code, not build artifacts like executables. If you want to provide a build of your software to users, look at [making a release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases). Release artifacts can be up to 2 GB.

If you want to upload the executable to the repository anyway, using LFS as @matej-kaska said is your only option.
